x-pack/solutions/security/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/index.ts

```diff
@@ -87,10 +87,8 @@ export const callAssistantGraph: AgentExecutor<true | false> = async ({
             // prevents the agent from retrying on failure
             // failure could be due to bad connector, we should deliver that result to the client asap
             maxRetries: 0,
-            metadata: {
-              connectorTelemetry: {
```

Comment:
It does get mapped like this in the chat model: https://github.com/elastic/kibana/blob/912f83088db3f562e5e2aedfd8c13f88df0ff890/x-pack/platform/packages/shared/ai-infra/inference-langchain/src/chat_model/inference_chat_model.ts#L191

Not sure why I thought it needed to be passed in like this though.